src/ChunkDef.h

```diff
@@ -112,7 +132,7 @@ class cChunkDef
 	/** Converts relative block coordinates into absolute coordinates with a known chunk location */
 	inline static Vector3i RelativeToAbsolute(const Vector3i & a_RelBlockPosition, int a_ChunkX, int a_ChunkZ)
 	{
-		return Vector3i(a_RelBlockPosition.x + a_ChunkX * Width, a_RelBlockPosition.y, a_RelBlockPosition.z + a_ChunkZ * Width);
+		return {a_RelBlockPosition.x + a_ChunkX * Width, a_RelBlockPosition.y, a_RelBlockPosition.z + a_ChunkZ * Width};
```

Comment:
I went ahead and changed it in the major files this PR already touches without any hitches (CI's still running, though), I didn't want to expand the scope of the PR to touch every instance in the codebase.